test/lerarenbeurs-scenarios.spec.js

```diff
@@ -504,4 +504,63 @@ describe('discipl-law-reg in scenarios with lerarenbeurs', () => {
       'DISCIPL_FLINT_PREVIOUS_CASE': actionLink3
     })
   }).timeout(5000)
+
+  it('should be able to get available actions after fact created from another action', async () => {
+    const needSsid = await core.newSsid('ephemeral')
+    await core.allow(needSsid)
+
+    const needLink = await core.claim(needSsid, {
+      'need': {
+        'act': '<<aanvraagformulieren lerarenbeurs verstrekken>>',
+        'DISCIPL_FLINT_MODEL_LINK': modelLink
+      }
+    })
+
+    const bestuursorgaanFactresolver = (fact) => {
+      if (typeof fact === 'string') {
+        return fact === '[template voor aanvraagformulieren studiekosten]' || fact === '[minister van Onderwijs, Cultuur en Wetenschap]'
+      }
+      return false
+    }
+
+    const actionLink = await lawReg.take(ssids['bestuursorgaan'], needLink, '<<aanvraagformulieren verstrekken voor subsidie studiekosten op de website van de DUO>>', bestuursorgaanFactresolver)
+
+    const availableActs = await lawReg.getAvailableActs(actionLink, ssids['belanghebbende'], [], [])
+
+    expect(availableActs).to.deep.equal([])
+  }).timeout(5000)
+
+  it('should be able to get potential actions after fact created from another action', async () => {
+    const needSsid = await core.newSsid('ephemeral')
+    await core.allow(needSsid)
+
+    const needLink = await core.claim(needSsid, {
+      'need': {
+        'act': '<<aanvraagformulieren lerarenbeurs verstrekken>>',
+        'DISCIPL_FLINT_MODEL_LINK': modelLink
+      }
+    })
+
+    const bestuursorgaanFactresolver = (fact) => {
+      if (typeof fact === 'string') {
+        return fact === '[template voor aanvraagformulieren studiekosten]' || fact === '[minister van Onderwijs, Cultuur en Wetenschap]'
+      }
+      return false
+    }
+
+    const actionLink = await lawReg.take(ssids['bestuursorgaan'], needLink, '<<aanvraagformulieren verstrekken voor subsidie studiekosten op de website van de DUO>>', bestuursorgaanFactresolver)
+
+    const availableActs = await lawReg.getPotentialActs(actionLink, ssids['belanghebbende'], [], [])
+
+    expect(availableActs).to.deep.equal([
+      {
+        'act': '<<indienen verzoek een besluit te nemen>>',
+        'link': 'link:discipl:ephemeral:ixZjrZnIHG8jqXH0mAnY9mm0irwaYWnfxu6qoXa9v4CvOfLEU+awWzWlT/mtmNwECbsOJJJKFeizbmytcpzVTw=='
```

Comment:
Having this link here hardcoded means that there's a hard dependency on the implementation of ephemeral. I tend to do one of two things here:
- Transform the available acts to an array with just the 'act' properties and test that.
- Retrieve the model using the model link and look up the relevant links.